src/NuGetGallery/Services/UserService.cs

```
@@ -40,6 +40,83 @@ public class UserService : IUserService
             EntitiesContext = entitiesContext;
         }
 
+        public async Task<Membership> AddOrUpdateMemberAsync(Organization organization, string memberName, bool isAdmin)
+        {
+            organization = organization ?? throw new ArgumentNullException(nameof(organization));
+            if (string.IsNullOrEmpty(memberName))
+            {
+                throw new ArgumentException(Strings.OrganizationMemberNameIsRequired, nameof(memberName));
```

`ArgumentException` isn't an `EntityException`, so I believe this will cause a 500 in the controller. Same with `DeleteMemberAsync`. Should be a 400.

---

Removed memberName check, will let it throw the EntityException below when no user is  found.